MapboxCoreNavigation/RouteController.swift

```diff
@@ -394,6 +397,7 @@ extension RouteController {
         if let lastReroute = outstandingFeedbackEvents.map({$0 as? RerouteEvent }).last {
             lastReroute?.update(newRoute: routeProgress.route)
         }
+        recentDistancesFromManeuver.removeAll()
```

Comment:
Does it make a difference if this is in `willReroute(notification:)` or `didReroute(notification:)`? Wondering if there’s a possibility of a race condition if a location even more distant from the maneuver comes in while waiting for a response from the API.